azure-cs-appservice/SharedAccessSignature.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Pulumi;
+using Pulumi.Azure.Storage;
+using Pulumi.Azure.Storage.Inputs;
+
+ public static class SharedAccessSignature
```

Comment:
Can he sneak in this class into the SDK? :)

Reply:
Yes, that's in the list https://github.com/pulumi/pulumi/issues/3406

Reply:
Brilliant